content/index.html

```diff
@@ -34,7 +34,29 @@ <h1 class="text-color text-center animated fadeInDown delay1">Nextflow</h1>
                     <a href="#Features" class="btn btn-color btn-xxl scroll">Find out more</a>
                 </p>
             </div>
-
+        </div>
+    </div>
+    <div class="container banner animated fadeInDown">
+        <div class="row">
+            <div class="col-sm-6" style="padding: 0;">
+                <a href="https://summit.nextflow.io">
+                    <img src="img/summit-2023.png" alt="Nextflow SUMMIT 2023">
+                </a>
+            </div>
+            <div class="col-sm-6 banner-rhs">
+                <h2 class="text-color">Nextflow SUMMIT 2023</h2>
+                <p>
+                    Join us for the annual community summit - this year we have two events!
+                </p>
+                <ul>
+                    <li><strong>Barcelona</strong>: October 16-20 <small><em>(in person &amp; virtual)</em></small></li>
+                    <li><strong>Boston</strong>: November 26-28 <small><em>(in person only)</em></small></li>
+                </ul>
+                <p>
+                    Networking, inspiring talks and engaging nf-core hackathons!
+                </p>
+                <a href="https://summit.nextflow.io" class="btn btn-color btn-xxl" style="padding-bottom:15px">Register now</a>
```

Comment:
margin, not padding 🙈 
```suggestion
                <a href="https://summit.nextflow.io" class="btn btn-color btn-xxl" style="margin-bottom:15px">Register now</a>
```